css/project-list-pages.styl

```diff
@@ -43,6 +43,10 @@
     .pill-button
       margin-right: 10px
       padding: 1em
+      color: rgb(64,64,64)
+      font-weight: normal
+      border-color: rgb(64,64,64) !important
```

Comment:
Bottom line, I spent so long trying to solve this problem, I can't afford to spend any more time on it. Something very odd is going on with the CSS inheritance on this page and so this is the best quick fix I can do for now. Technically it's probably not necessary and there's a better way. Pragmatically, it's necessary for now.